.circleci/config.base.yml

```diff
@@ -1377,7 +1373,7 @@ commands:
                 command: |
                   cd out
                   ln -sf amplify-pkg-linux-x64 amplify
-                  echo "export PATH=$AMPLIFY_DIR:$PATH" >> $BASH_ENV
+                  echo "export PATH=$AMPLIFY_DIR:$PATH:/home/circleci/repo/node_modules/.bin/" >> $BASH_ENV
```

Comment:
what do we need from `/home/circleci/repo/node_modules/.bin/` ?

Reply:
yarn no longer has a global bin directory, we used to set our amplify binary there, I changed our package.json scripts to add it to a .bin directory in the project root, that .bin directy needs to be in path for our tests to find the amplify binary